app/league/[leagueId]/entries/[entryId]/week/Week.tsx

```diff
@@ -28,12 +31,22 @@ import { ISchedule } from './WeekTeams.interface';
 // eslint-disable-next-line no-unused-vars
 const Week = ({ league, NFLTeams, week }: IWeekProps): JSX.Element => {
   const [schedule, setSchedule] = useState<ISchedule[]>([]);
+  const [selectedLeague, setSelectedLeague] = useState<ILeague | undefined>();
```

Comment:
Should the state be initialized to string? i.e.:  `const [selectedLeague, setSelectedLeague] = useState<string>('');`?

Reply:
I'm gonna leave this to @chris-nowicki, since he helped me with this by implementing it Though, initializing it to a string makes sense to me!